apps/tailwind-components/components/input/Array.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <div>
+    <div
+      v-for="(value, index) in valuesNotUndefined"
+      :key="index"
+      class="flex items-center gap-1"
+    >
+      <Input
+        :id="id + '_' + index"
+        :modelValue="valuesNotUndefined[index]"
+        v-bind="partialProps($props)"
+        :type="nonArrayType(props.type || 'STRING_ARRAY')"
+        @blur="emit('blur')"
+        @focus="emit('focus')"
+        @update:model-value="
+          emit('update:modelValue', setValues($event, index))
+        "
+      />
+      <Button
+        iconOnly
+        type="secondary"
+        icon="trash"
+        label="Remove item"
+        v-if="valuesNotUndefined.length > 1"
+        @click="clearInput(valuesNotUndefined, index)"
+      />
+    </div>
+    <Button
+      type="secondary"
+      size="small"
+      icon="plus"
+      label="Add a additional item"
+      @click="addItem(valuesNotUndefined)"
+    />
+  </div>
+</template>
+
+<script setup lang="ts">
+import type { IInputProps } from "~/types/types";
+import type { CellValueType } from "../../../metadata-utils/src/types";
+
+let props = defineProps<
+  IInputProps & {
+    modelValue: any[] | undefined;
+    type?: string;
+  }
+>();
+
+const values = ref(props.modelValue);
```

Comment:
I think this breaks reactivity. I think the way to go here is either use defineModel() or use const {....} = toRefs(props)